buildpacks/buildpack-clojure/tests/clojure-ring/Procfile

```diff
@@ -1 +1 @@
-web: lein trampoline run
+web: lein trampoline run -m helloworld.web
```

Comment:
@iSDP out of curiosity, why was this change needed?

Reply:
looked at the docs, showed that flag so added it.